scripts/basic-test.sh

```diff
@@ -46,15 +46,16 @@ if [ "$DARWIN" = true ]; then
 fi
 
 # fake-factory doesn't have a correct universal wheel
-pip install --no-use-wheel .[fakefactory]
+pip install --no-use-wheel faker
```

Comment:
I’m not sure what version of pip runs on Travis, but when I run this command with pip 9.0.1, I get a deprecation warning:

```
DEPRECATION: --no-use-wheel is deprecated and will be removed in the future.  Please use --no-binary :all: instead.
```

Perhaps worth tweaking the command while we’re here?